tests/validate_templates.sh

```diff
@@ -3,6 +3,7 @@
 # The 'root_dir' and 'environment_deployments' variables
 # should be set when we source this.
 
+FAIL=0
```

Comment:
Could we move this to `tests/test_edx_east_roles.sh` to keep this file unforked for now?